command.go

```diff
@@ -337,7 +338,9 @@ func (c *Command) RunParallel(ctx context.Context) error {
 			if !c.ignoreError {
 				catcher.Add(err)
 			}
+			c.procs = append(c.procs, <-procs...)
```

Comment:
it feels like this is probably a recipe for deadlocks, particularly if there's a panic

Reply:
See previous comment.